src/main/java/com/synopsys/integration/alert/channel/jira/cloud/JiraCloudRequestDelegator.java

```diff
@@ -98,4 +92,17 @@ public IssueTrackerResponse sendRequests(List<IssueTrackerRequest> requests) thr
         return jiraIssueHandler.createOrUpdateIssues(validIssueConfig, requests);
     }
 
+    private void checkIfAlertPluginIsInstalled(JiraCloudServiceFactory jiraCloudServiceFactory, JiraCloudProperties jiraProperties) throws IssueTrackerException {
```

Comment:
Should this be passed PluginManagerService, username, and accessToken?

Reply:
It's not a public method, so I think it's better to minimize the clutter. Especially since this operation won't always happen.